sdk/quota/arm-quota/samples-group-quota/AddSubscriptionGroupQuota.js

```diff
@@ -19,10 +19,9 @@ require("dotenv").config();
  * x-ms-original-file: specification/quota/resource-manager/Microsoft.Quota/preview/2023-06-01-preview/examples/GroupQuotasSubscriptions/PutGroupQuotasSubscription.json
  */
 async function groupQuotaSubscriptionsPutSubscriptions() {
-  const subscriptionId =
-    process.env["QUOTA_SUBSCRIPTION_ID"] || "00000000-0000-0000-0000-000000000000";
-  const managementGroupId = "E7EC67B3-7657-4966-BFFC-41EFD36BAA09";
-  const groupQuotaName = "groupquota1";
+  const subscriptionId ="65a85478-2333-4bbd-981b-1a818c944faf";
+  const managementGroupId = "testMgIdRoot";
+  const groupQuotaName = "sdk-test-group-quota";
```

Comment:
Since this is auto-generated, could we update the swagger examples directly?

https://github.com/Azure/azure-rest-api-specs/tree/69a1ccd8f036643aa4c9aa6d8b64f754759ac1d9/specification/quota/resource-manager/Microsoft.Quota/preview/2023-06-01-preview

Reply:
This would kick off another swagger review afaik. As mentioned before, while we transition our PUT APIs to POST, to temporarily unblock customers we have updated these samples so they can achieve the same functionality as the CreateOrUpdateAndWait without it breaking.

Reply:
I would suggest we do this parallelly, otherwise this would be overried next time.